.drop.sh

```diff
@@ -26,11 +28,14 @@ CONFIGPRODUCTION="$(cat src/config/production.ts | sed "s/oldestVersion:.*/oldes
 CONFIGSTAGING="$(cat src/config/staging.ts | sed "s/latestVersion:.*/latestVersion: '${VERSION}',/;s/oldestVersion:.*/oldestVersion: '${LASTVERSION}'/")" && echo "${CONFIGSTAGING}" > src/config/staging.ts
 CONFIGTEST="$(cat src/config/test.ts | sed "s/latestVersion:.*/latestVersion: '${VERSION}',/;s/oldestVersion:.*/oldestVersion: '${LASTVERSION}'/")" && echo "${CONFIGTEST}" > src/config/test.ts
 CONFIGBOOTSTRAP="$(cat src/config/bootstrap.ts | sed "s/oldestVersion:.*/oldestVersion: '${LASTVERSION}'/")" && echo "${CONFIGBOOTSTRAP}" > src/config/bootstrap.ts # Bootstrap doesn't get the latest version edited
+CONFIGDOCSVERSION="$(cat docs/versions.json | sed "/${OLDESTVERSION}/d")" && echo "${CONFIGDOCSVERSION}" docs/versions.json
+CONFIGDOCSVERSIONVALID="$(cat docs/versions.json | sed "s/,]/]/")" && echo "${CONFIGDOCSVERSIONVALID}" docs/versions.json
```

Comment:
Why is this split into two lines? Why not just:

```suggestion
CONFIGDOCSVERSION="$(cat docs/versions.json | sed "/${OLDESTVERSION}/d" | sed "s/,]/]/")" && echo "${CONFIGDOCSVERSION}" docs/versions.json
```